profcond.php

```diff
@@ -31,8 +31,8 @@ function profcond_civicrm_buildForm($formName, &$form) {
     $pageConfig = array_merge_recursive($pageConfig, CRM_Utils_Array::value($pageId, $config[$useConditionals], []));
 
     if ($priceSetId) {
-      $pageConfig = array_merge_recursive($pageConfig, ($config['priceset']['all'] ?: []));
-      $pageConfig = array_merge_recursive($pageConfig, ($config['priceset'][$priceSetId] ?: []));
+      $pageConfig = array_merge_recursive($pageConfig, (isset($config['priceset']['all']) ? $config['priceset']['all'] : []));
+      $pageConfig = array_merge_recursive($pageConfig, (isset($config['priceset'][$priceSetId]) ? $config['priceset'][$priceSetId] : []));
```

Comment:
@lucky091588 CiviCRM core has recently started using the `??` operator (https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op) in cases like this. Please use that here instead of ternary `?:`. Thanks.